### PR TITLE
Udating code and adding company, finance and loan application page

### PR DESCRIPTION
**Pull request by Jeevan1**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -51,13 +53,19 @@ const App = () => {
...
+                <Route path="/company" exact element={<CompanyPage />} />
+                <Route
+                  path="/loan-application"
+                  exact
+                  element={<LoanApplicationPage />}
+                />
                 <Route path="/users/:id" exact element={<UserProfile />} />
                 <Route path="/application" exact element={<LoanForm />} />
-                <Route path="/loans" exact element={<LoansPage />} />
+                {/* <Route path="/loans" exact element={<LoansPage />} />
```
Comment (RahulRimal) [inline src/App.jsx:65]: Also implement loans page

File: src/App.jsx [hunk 2]
```diff
@@ -51,13 +53,19 @@ const App = () => {
...
 
                 {/* <Route path="/play" exact element={<Playground />} /> */}
-                <Route path="/loans" exact element={<LoansPage />} />
+                {/* <Route path="/loans" exact element={<LoansPage />} />  */}
```
Comment (RahulRimal) [inline src/App.jsx:68]: remove commented code

**Review by RahulRimal — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (Jeevan1) [reply src/App.jsx]: done

File: src/App.jsx (hunk 2 above)
Comment (Jeevan1) [reply src/App.jsx]: done

**Review by RahulRimal — APPROVED** [review]

**Pull request merged**